<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>anitools::widgets::AffineTransformationList</class>
 <widget class="QWidget" name="anitools::widgets::AffineTransformationList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="anitools::widgets::WidgetList" name="mWidgetList" native="true"/>
   </item>
   <item>
    <widget class="QToolButton" name="mButtonAppend">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton
{
	border:0;
	color:palette(link);
}
QToolButton:hover
{
	text-decoration: underline;
}</string>
     </property>
     <property name="text">
      <string>Append New Transformation</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>anitools::widgets::WidgetList</class>
   <extends>QWidget</extends>
   <header>widgetlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
